src/cli/mod.rs

```diff
@@ -116,36 +160,95 @@ struct ReplCli {
     #[clap(long, value_parser)]
     zstore: Option<PathBuf>,
 
+    #[clap(long, value_parser)]
+    config: Option<PathBuf>,
+
     #[clap(long, value_parser)]
     limit: Option<usize>,
-    // #[clap(long, value_parser)]
-    // rc: Option<usize>,
+
+    #[clap(long, value_parser)]
+    rc: Option<usize>,
+
+    #[clap(long, value_parser)]
+    field: Option<String>,
+
+    #[clap(long, value_parser)]
+    backend: Option<String>,
 }
 
 impl ReplArgs {
     pub fn into_cli(self) -> ReplCli {
         ReplCli {
             load: self.load,
             zstore: self.zstore,
+            config: self.config,
             limit: self.limit,
-            // rc: self.rc,
+            rc: self.rc,
+            field: self.field,
+            backend: self.backend,
         }
     }
 }
 
-fn get_field() -> Result<LanguageField> {
-    if let Ok(lurk_field) = std::env::var("LURK_FIELD") {
-        match lurk_field.to_lowercase().as_str() {
-            "bls12-381" => Ok(LanguageField::BLS12_381),
-            "pallas" => Ok(LanguageField::Pallas),
-            "vesta" => Ok(LanguageField::Vesta),
-            _ => bail!("Field not supported: {lurk_field}"),
-        }
-    } else {
-        Ok(DEFAULT_FIELD)
+fn parse_field(field_str: &String) -> Result<LanguageField> {
+    match field_str.to_lowercase().as_str() {
+        "pallas" => Ok(LanguageField::Pallas),
+        "vesta" => Ok(LanguageField::Vesta),
+        "bls12-381" => Ok(LanguageField::BLS12_381),
+        _ => bail!("Field not supported: {field_str}"),
+    }
+}
+
+fn parse_backend(backend_str: &String) -> Result<Backend> {
+    match backend_str.to_lowercase().as_str() {
+        "nova" => Ok(Backend::Nova),
+        "groth16" => Ok(Backend::Groth16),
```

Comment:
Despite our ubiquitous erroneous type names, `Groth16` is a related but different proving system from what the user will actually get (SnarkPack).

Reply:
SnarkPack+ then, please.